src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Expected/MixedMode_x64_VS2019_CPlusPlus_DEBUG_DEFAULT.dll.sarif

```diff
@@ -69,27 +69,6 @@
             }
           ]
         },
-        {
-          "ruleId": "BA2004",
-          "ruleIndex": 2,
-          "message": {
-            "id": "Warning_NativeWithInsecureStaticLibraryCompilands",
-            "arguments": [
-              "MixedMode_x64_VS2019_CPlusPlus_DEBUG_DEFAULT.dll",
-              "Microsoft (R) Optimizing Compiler : cxx : 19.29.30133.0 : [directly linked] [No hash value present] (.NETFramework,Version=v4.0.AssemblyAttributes.obj)\r\n"
```

Comment:
In this test you can see user provided `MD5`, while the system generated files are with `No hash`. 
No need to check the system generated files and show the info that user not able to change.